.gitlab-ci.yml

```diff
@@ -0,0 +1,48 @@
+image: python:3.6
+
+cache:
+  paths:
+  - chalice/.chalice/venv
+  - daemons/dss-sync/.chalice/venv
+  - daemons/dss-index/.chalice/venv
+
+variables:
+  ES_VERSION: 5.4.2
+  ES_DOWNLOAD_URL: https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-${ES_VERSION}.tar.gz
+  DSS_TEST_ES_PATH: ./elasticsearch-${ES_VERSION}/bin/elasticsearch
+  DSS_ES_TIMEOUT: 30
+  DSS_UNITTEST_OPTS: "-v"
+  TERRAFORM_APPLY_ARGS: "-auto-approve"
+
+stages:
+  - deploy_dev
+
+before_script:
+  - apt-get update -q -y
```

Comment:
Suggested: I'd prefer the long-forms of these switches. For readability.

Reply:
Done.